### PR TITLE
Added new fixtures and fixed bugs in existing ones.

### PR DESCRIPTION
**Pull request by kluszon**
Added new fixtures and fixed bugs in existing ones.

### CODE REVIEW

File: resources/fixtures/Cameo/Cameo-Auro-Spot-300.qxf [hunk 1]
```diff
@@ -139,7 +139,7 @@
  <Channel Name="Prism">
   <Group Byte="0">Prism</Group>
   <Capability Min="0" Max="5" Preset="PrismEffectOff">Prism off</Capability>
-  <Capability Min="6" Max="10" Preset="PrismEffectOn" Res1="3">Prism in</Capability>
+  <Capability Min="6" Max="10" Preset="PrismEffectOn" Res1="0">Prism in</Capability>
```
Comment (joshuaurieli) [inline resources/fixtures/Cameo/Cameo-Auro-Spot-300.qxf:142]: @kluszon This looks like a regression to me. `Preset="PrismEffectOn" Res1="3">Prism in` is correct.

File: resources/fixtures/Robe/Robe-ColorSpot-250-AT.qxf [hunk 2]
```diff
@@ -155,15 +151,15 @@
   <Capability Min="248" Max="255">Macro 12</Capability>
  </Channel>
  <Channel Name="Prism Rotation">
-  <Group Byte="0">Prism</Group>
-  <Capability Min="0" Max="0" Preset="RotationStop">No Rotation</Capability>
-  <Capability Min="1" Max="127" Preset="RotationClockwiseFastToSlow">CW Rotation Fast to Slow</Capability>
-  <Capability Min="128" Max="129" Preset="RotationStop">No Rotation</Capability>
-  <Capability Min="130" Max="255" Preset="RotationCounterClockwiseSlowToFast">CCW Rotation Slow to Fast</Capability>
+  <Group Byte="0">Speed</Group>
```
Comment (joshuaurieli) [inline resources/fixtures/Robe/Robe-ColorSpot-250-AT.qxf:154]: @kluszon Please change this back to Prism

File: resources/fixtures/Robe/Robe-ColorSpot-250-AT.qxf [hunk 3]
```diff
@@ -138,9 +134,9 @@
  </Channel>
  <Channel Name="Gobo Indexing Fine" Preset="GoboIndexFine"/>
  <Channel Name="Prism">
-  <Group Byte="0">Prism</Group>
-  <Capability Min="0" Max="19" Preset="PrismEffectOff">Open hole</Capability>
-  <Capability Min="20" Max="159" Preset="PrismEffectOn" Res1="3">3-facet rotating</Capability>
+  <Group Byte="0">Beam</Group>
```
Comment (joshuaurieli) [inline resources/fixtures/Robe/Robe-ColorSpot-250-AT.qxf:137]: @kluszon Please change this back to Prism

**Comment by joshuaurieli** [comment]
@kluszon Looks good now. Thanks for the changes.

File: resources/fixtures/Generic/Generic-Generic-Smoke.qxf [hunk 4]
```diff
@@ -1,34 +1,36 @@
...
+ <Channel Name="Pump" Preset="IntensityMasterDimmer"/>
  <Channel Name="Fan">
-  <Group Byte="0">Intensity</Group>
-  <Capability Min="0" Max="255">Min &lt; Max</Capability>
+  <Group Byte="0">Speed</Group>
+  <Capability Min="0" Max="255" Preset="SlowToFast">Fan Speed, slow - fast</Capability>
  </Channel>
- <Mode Name="Normal">
+ <Channel Name="Fan 2" Preset="IntensityDimmer"/>
+ <Mode Name="1 Channel">
```
Comment (mcallegari) [inline resources/fixtures/Generic/Generic-Generic-Smoke.qxf:18]: Renaming modes means breaking existing projects. Please don't do that unless it's a definition mistake.

**Review by mcallegari — CHANGES_REQUESTED** [review]

File: resources/fixtures/Robe/Robe-Pointe.qxf [hunk 5]
```diff
@@ -219,7 +238,7 @@
  </Channel>
  <Channel Name="Dimmer" Preset="IntensityDimmer"/>
  <Channel Name="Dimmer fine" Preset="IntensityDimmerFine"/>
- <Mode Name="Main">
+ <Mode Name="Mode 1">
```
Comment (joshuaurieli) [inline resources/fixtures/Robe/Robe-Pointe.qxf:241]: Having mode in the name is considered an error by the fixturevalidator so this and similar changes is necessary.

**Comment by mcallegari** [comment]
Thanks. Who should I credit for this lot of fixtures in the changelog? @kluszon ? Ray Console? @joshuaurieli ?

**Comment by joshuaurieli** [comment]
Thanks. Please credit: @kluszon @joshuaurieli @claudia.dunkel

File: resources/fixtures/Robe/Robe-Pointe.qxf (hunk 5 above)
Comment (FloEdelmann) [reply resources/fixtures/Robe/Robe-Pointe.qxf:241]: @mcallegari @joshuaurieli @kluszon This comment is still not addressed, although the PR is now merged.

**Comment by mcallegari** [comment]
Merged.
There are still a lot of errors, but I'm gonna fix them myself.
Please remember next time not to commit generic fixtures (this is the second time I reject them)

File: resources/fixtures/Robe/Robe-Pointe.qxf (hunk 5 above)
Comment (mcallegari) [reply resources/fixtures/Robe/Robe-Pointe.qxf:241]: @FloEdelmann too fast! See my comment. I'm gonna fix errors myself.

**Comment by joshuaurieli** [comment]
Thanks for merging the devices. Also thank you for fixing your errors. Could you please post a link to your changes in this thread or include @joshuaurieli and @kluszon the the relevant pull request. We would like to do better next time.

What was wrong with the generic devices? Last time the comment was that they were missing physical properties which this time were included. Generic devices are very useful to users with simple devices to quickly get started.

**Comment by joshuaurieli** [comment]
@mcallegari What was wrong with the generic devices? Last time the comment was that they were missing physical properties which this time were included. Generic devices are very useful to users with simple devices to quickly get started.

**Comment by mcallegari** [comment]
> @mcallegari What was wrong with the generic devices? Last time the comment was that they were missing physical properties which this time were included. Generic devices are very useful to users with simple devices to quickly get started.

Generic fixtures can be achieved with generic dimmers and RGB panel. Also, generic definition will never have precise measures of a real existing fixture so the idea is to encourage users to create specific definitions for real products.

**Pull request merged**